project_task_tag/tests/test_project_tags.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.tests.common import TransactionCase
+
+
+class TestProjectTags(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.tag_1 = cls.env["project.tags"].create({"name": "tag1"})
```

Comment:
I'm not sure what you are testing here :thinking: 
As the change now is only in the view, I would use `odoo.tests.Form` to make sure the modifiers of the tag_ids field are properly set.

Reply:
sorry, it was after yesterdays suggestion but it had more sense to have tests when field was computed, i don't think it is crucial to test domain on the field.